bin/start-celery

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Starts a celery worker / heartbeat job. Must be run with a type of process: worker | beat
+
+set -e
+
+# this kills all processes when the last one terminates
+trap 'kill $(jobs -p)' EXIT
```

Comment:
Doesn't look like we need the jobs trap here?

Reply:
Indeed, fixed.